mmdet/core/anchor/anchor_generator.py

```diff
@@ -242,8 +242,8 @@ def single_level_grid_anchors(self,
         Args:
             base_anchors (torch.Tensor): The base anchors of a feature grid.
             featmap_size (tuple[int]): Size of the feature maps.
-            stride (tuple[int], optional): Stride of the feature map.
-                Defaults to (16, 16).
+            stride (tuple[int], optional): Stride of the feature map in order
```

Comment:
We may also update the docstring in the AnchorGenerator's arguments at around line 15.